### PR TITLE
feat:add three lecture blocks

### PR DESCRIPTION
**Pull request by igortrsk**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1098,13 +1133,35 @@ <h3 class="text-4xl lg:text-5xl text-center text-white">
...
-                    class="order-first sm:order-last flex items-center gap-10 pb-14 sm:pb-0"
+                    class="order-first sm:order-last flex max-[315px]:flex-col items-center gap-10 pb-14 sm:pb-0"
                 >
                     <a
                         class="text-lg lg:hover:text-brand-green transition ease-linear"
                         href="mailto:day@zkwarsaw.dev"
                         >Contact Us</a
                     >
+
+                    <!-- facebook -->
```
Comment (OrestTa) [inline public/index.html:1144]: pls try to use separate PRs for separate features in the future

File: public/index.html [hunk 2]
```diff
@@ -555,7 +590,7 @@ <h5 class="font-bold text-2xl text-white">Speakers</h5>
   -->
 
             <!-- schedule -->
-            <!--
+            <!--
```
Comment (OrestTa) [inline public/index.html:593]: intentional?

File: public/index.html [hunk 3]
```diff
@@ -347,15 +348,49 @@
...
+                <!-- pb-8 pt-6 md:pt-28 -->
+                <div
+                    class="flex justify-center flex-col py-10 md:py-12 lg:py-16 px-5"
+                >
+                    <div class="max-w-4xl mx-auto pb-4 md:pb-6 lg:pb-10 px-5">
+                        <h3 class="text-4xl lg:text-5xl text-center text-white">
+                            Unveil Zero-Knowledge Proofs through 3 expert-led
+                            blocks: Beginner, Math, and Tech. Discover secure
+                            data privacy's secrets with our captivating lectures
+                            and workshops.
```
Comment (OrestTa) [inline public/index.html:370]:
```suggestion
                            Event for ZKP researchers, hackers, and builders. Learn all about zero-knowledge proofs through three focussed blocks of lectures and workshops:
```

**Review by OrestTa — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (igortrsk) [reply public/index.html:1141]: Sure, thanks

File: public/index.html [hunk 4]
```diff
@@ -364,10 +364,7 @@ <h3 class="text-4xl lg:text-5xl text-center text-white">
                 >
                     <div class="max-w-4xl mx-auto pb-4 md:pb-6 lg:pb-10 px-5">
                         <h3 class="text-4xl lg:text-5xl text-center text-white">
-                            Unveil Zero-Knowledge Proofs through 3 expert-led
-                            blocks: Beginner, Math, and Tech. Discover secure
-                            data privacy's secrets with our captivating lectures
-                            and workshops.
+                            Event for ZKP researchers, hackers, and builders. Learn all about zero-knowledge proofs through three focussed blocks of lectures and workshops:
```
Comment (OrestTa) [inline public/index.html:367]:
```suggestion
                            Event for ZKP researchers, hackers, and builders. Learn all about zero-knowledge proofs through three focussed blocks of lectures and workshops.
```

**Review by OrestTa — DISMISSED** [review]

**Comment by OrestTa** [comment]
@igortrsk rdy for merge? wanna resolve the conflicts?

**Review by marekkirejczyk — DISMISSED** [review]

**Review by OrestTa — APPROVED** [review]

**Pull request merged**